ui/src/table.test.tsx

```diff
@@ -360,6 +360,76 @@ describe('Table.tsx', () => {
       expect(emitMock).toHaveBeenCalledTimes(1)
     })
 
+    it('Does not fire select event - single selection - search not matching text', () => {
```

Comment:
Is the "no match" significant? It shouldn't fire select event whether search/filter matches or not.